src/features/classification/Classification.js

```diff
@@ -89,7 +91,15 @@ const Classification = ({
                     <p className="bdl-Classification-modifiedBy" data-testid="classification-modifiedby">
                         <FormattedMessage
                             {...messages.modifiedBy}
-                            values={{ modifiedAt: formattedModifiedAt, modifiedBy }}
+                            values={{
+                                modifiedActionLabel: useAppliedByLabel ? (
+                                    <FormattedMessage {...messages.appliedByText} />
+                                ) : (
+                                    <FormattedMessage {...messages.importedFromText} />
```

Comment:
The action label is by default going to be `Applied by` (except in the instance it's by MIP which we will use `Imported from`) -- wondering if we'd prefer explicit individual flags like `useAppliedByLabel` `useImportedByLabel` and render it outside of this `FormattedMessage` as {`useAppliedByLabel && <FormattedMessage {...messages. appliedByText} />` or have an optional `useImportedByLabel` and fall back to `Applied By` when that optional flag is not existence.

Reply:
I think this could probably be addressed with the prop name suggestion above. I'd say it's preferable to have a single prop since we will only be rendering one or the other.